modules/inception/gcp/variables.tf

```diff
@@ -21,6 +21,7 @@ variable "network_prefix" {
 variable "inception_sa" {}
 variable "tf_state_bucket_name" {}
 variable "buckets_location" {}
+variable "policy_data_override" { default = null }
```

Comment:
lets use a better name. `tf_state_bucket_policy` for example. `policy_data_override` isn't clear.